src/index.js

```diff
@@ -355,31 +359,35 @@ function createConfig(options, entry, format, writeMeta) {
 									jsx: options.jsx,
 								},
 							},
+							tsconfigOverride: {
+								compilerOptions: {
+									target: 'es2017',
+								},
+							},
 						}),
 					!useTypescript && flow({ all: true, pretty: true }),
-					nodent({
+					// Only used for async await
+					babel({
+						babelrc: false,
```

Comment:
maybe add a comment on why we are doing this? I can imagine someone opening a PR to put this on true.